src/refmt/refmt2.re

```diff
@@ -238,13 +240,12 @@ let refmt = (code, inLang, inType, outLang) => {
       | Ast(REOI(_)) => REO
       | Error(_) => UnknownLang
       };
-    (trueIn, trueOut, printedResult)
+    Js.log2("hdon sez: refmt2 is okay", printedResult);
+    Result.Ok((trueIn, trueOut, printedResult))
   } {
-  | a => (UnknownLang, UnknownLang, Obj.magic(a))
+  | a => {
+    Js.log("hdon sez: refmt2 is error");
+    Result.Error(a |> Printexc.to_string)
```

Comment:
This doesn't seem to be giving very useful responses. Everything gets printed as `Failure(_)` when I try. I think this can be fixed in a followup.

Reply:
Yeah it's definitely got some limitations. I was hoping to get some feedback about it.

Reply:
Yeah I think `Printexc.print` gives more control over the actual output here so we can make sure things look nice.